src/meshapi/models/los.py

```diff
@@ -38,6 +38,9 @@ class LOSSource(models.TextChoices):
     )
 
     analysis_date = models.DateField(
+        default=None,
+        blank=True,
+        null=True,
```

Comment:
I know this has something to do with the unreliable data we have about our existing links, but why would we want this to be nullable? What was the impetus for this decision?

Reply:
This boils down to basically the same discussion we had about whether or not to make email address nullable since it's so core to our processes (can't find the issue, maybe it was on a call?) 

The reality is that for some of these LOS entries, there is a small chance we just don't have a date on the upstream Link object that they have been generated from. We can make up a bullshit date in these cases (1970-01-01 or the founding date of the mesh or something), but to me it feels better to pass the `null` along to the consumer so that they can make an educated choice (and not do silly things with our silly date). Fundamentally if we don't have the data we don't have the data

Within meshdb this field isn't currently used for anything, we just carry it along as metadata. Likely future use cases would be things like filtering out old LOS analyses when trying to asses LOS impacts of a new building or something

Perhaps we should actually just do `null=True`, without setting `default` or `blank`. This would force admins who try to create/modify LOS objects with admin forms to always provide a date. Though this includes cases where they modify an auto-generated object that did not have a date before they touched it. That might be annoying on the one hand, but rare, and on the other hand they can probably make a higher quality guess than the automated system can

Reply:
Discussed on a call: we're going to keep it not required in the admin forms, but on the create form, auto fill the current date